verify/cert.go

```diff
@@ -10,28 +10,29 @@ import (
 
 	"github.com/Layr-Labs/eigenda-proxy/common/consts"
 	"github.com/Layr-Labs/eigenda/api/grpc/disperser"
-	binding "github.com/Layr-Labs/eigenda/contracts/bindings/EigenDAServiceManager"
+	binding "github.com/Layr-Labs/eigenda/contracts/bindings/EigenDACertVerifier"
+	svc_binding "github.com/Layr-Labs/eigenda/contracts/bindings/EigenDAServiceManager"
```

Comment:
```suggestion
	edsm_binding "github.com/Layr-Labs/eigenda/contracts/bindings/EigenDAServiceManager"
```

Reply:
do you mind using this convention? its more precise and its what we use everywhere in the middleware contracts repo

Reply:
[20208b9](https://github.com/Layr-Labs/eigenda-proxy/pull/264/commits/20208b9ad29957cbefb39af4d314765ffde423c7)